cpp/tests/c_api/uniform_neighbor_sample_test.c

```diff
@@ -20,11 +20,351 @@
 #include <cugraph_c/graph.h>
```

Comment:
I don't see any tests for updates to the tests in `mg_uniform_neighbor_sample_test.c`

Reply:
Again, that's a shortcoming of PR #3696.  I'll take a look at this in that PR.

Reply:
Added tests to PR #3696